.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
**question**: Should we remove this file from the repository?

I don't know what `uiDesigner.xml` is, but I have a feeling we don't need to include it in version control. Perhaps we could do a little research to understand its purpose and make a decision?